go/130/developer-help-mobile/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Can we do redirects with web.config instead? I'm trying to move non-content like that into web.config as it's easier to maintain over time

Reply:
Refactored to web.config.  

Also added 12.0 to the archive download links